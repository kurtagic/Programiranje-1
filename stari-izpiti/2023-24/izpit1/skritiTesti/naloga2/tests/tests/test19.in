1 35 87 603
26 79 629
11 42 206
11 26 414
5 65 302
34 26 512
29 77 134
5 11 396
32 30 200
24 26 594
26 26 511
26 77 336
26 11 234
3 55 191
15 84 54
29 26 500
20 77 306
15 29 451
16 76 448
22 26 158
3 76 64
13 30 336
25 39 61
19 77 42
19 21 570
4 11 52
33 36 365
32 4 58
32 38 596
19 22 146
17 37 365
12 42 536
16 3 339
7 38 180
19 25 280
32 12 153
24 30 58
13 11 616
21 70 315
32 50 510
7 11 630
15 54 55
25 21 68
20 8 201
22 40 30
5 17 404
12 21 133
20 40 381
20 37 549
0 48 211
3 29 187
26 49 153
19 65 611
23 34 92
3 49 80
15 1 258
7 54 106
2 58 618
34 14 632
13 84 92
18 50 76
21 85 434
25 79 275
27 82 125
24 77 356
23 49 421
28 57 33
25 44 597
21 25 476
24 5 348
24 58 309
26 81 151
22 40 236
13 10 622
24 67 286
26 86 489
25 11 262
20 79 286
6 77 118
14 45 185
13 65 168
31 76 321
10 56 546
34 44 365
19 76 89
7 18 586
20 6 561
14 55 571
4 24 276
11 33 154
5 22 103
7 77 178
11 43 574
10 15 280
14 55 424
3 19 252
31 36 245
13 30 551
32 81 409
13 33 315
3 84 75
6 34 301
15 21 446
14 63 605
17 45 183
31 67 565
3 78 145
27 78 474
32 38 425
23 45 340
18 42 212
27 49 306
13 11 141
26 82 344
3 11 171
19 76 413
13 44 585
15 34 531
20 73 191
13 81 619
7 46 587
4 42 417
29 1 454
2 79 507
26 0 156
22 33 148
34 24 557
2 42 586
12 24 458
19 77 627
5 1 97
6 76 76
25 67 537
25 69 547
28 11 432
20 13 481
27 9 448
3 86 582
17 68 293
17 0 592
2 83 81
14 30 518
21 9 138
18 62 45
30 1 506
4 29 113
8 49 17
21 72 369
5 12 221
8 29 475
33 2 492
34 26 281
17 41 553
11 62 62
33 3 254
2 24 164
33 48 286
25 45 456
22 11 316
14 79 123
12 67 468
8 67 625
16 73 123
17 50 182
34 80 345
23 16 15
34 53 523
18 0 162
34 54 479
30 15 159
33 8 280
24 49 296
22 22 543
11 67 352
34 80 574
15 35 292
5 48 19
10 42 460
7 33 273
24 64 493
14 19 2
2 48 388
6 5 556
25 18 131
7 31 56
3 26 494
23 5 338
15 76 298
17 75 590
4 29 611
7 7 340
26 12 182
7 39 533
19 10 626
10 42 585
17 5 584
8 34 384
30 4 281
14 42 532
15 52 318
17 85 484
15 82 264
7 57 316
24 44 104
4 4 115
13 46 461
20 86 111
24 2 550
13 0 243
23 78 407
3 85 268
30 54 296
15 30 347
24 76 53
17 54 105
4 64 276
27 67 130
23 25 209
26 63 253
15 25 323
6 67 65
16 64 140
16 68 324
0 38 245
25 69 3
17 80 565
4 6 538
5 25 461
0 79 179
3 32 194
29 1 40
12 59 608
15 46 62
7 33 488
3 2 547
27 67 291
29 22 195
17 69 628
30 12 450
7 54 188
18 39 428
15 20 126
31 62 600
28 7 408
12 9 594
19 17 584
9 2 479
34 40 72
27 36 407
9 82 451
9 68 632
24 86 381
7 9 518
7 62 642
5 76 478
7 41 82
25 62 516
29 34 512
25 71 315
18 79 587
14 54 292
19 19 104
2 19 615
1 17 216
24 5 580
27 65 25
7 43 18
34 84 524
6 73 506
22 70 588
10 36 398
14 82 245
15 42 345
31 70 229
22 54 251
5 42 47
19 52 407
14 20 247
29 0 384
23 10 95
30 4 339
10 9 406
32 80 537
5 4 408
19 81 26
14 78 495
33 84 20
13 22 160
34 58 214
6 53 57
32 24 266
15 42 397
28 38 424
22 14 429
31 50 378
0 11 253
24 77 470
19 49 281
30 56 138
28 12 98
23 26 75
33 50 467
8 25 280
13 15 555
13 4 404
22 15 641
5 71 255
25 43 496
1 78 631
10 65 42
33 64 616
10 60 566
9 84 301
22 18 583
5 86 468
34 7 203
19 35 577
6 14 91
0 40 465
3 31 257
0 30 466
12 15 190
28 37 217
11 43 442
10 10 353
18 22 275
14 16 104
34 64 538
11 32 313
19 11 488
12 24 145
15 39 532
13 36 600
33 0 349
28 11 248
24 55 339
16 21 303
23 75 5
20 56 188
15 4 79
28 57 42
8 42 314
14 82 338
15 29 120
8 61 627
27 49 319
13 29 228
18 62 558
5 71 351
19 0 559
13 0 253
24 21 248
19 83 179
0 65 181
26 50 557
12 58 6
14 86 368
31 24 631
32 7 435
14 85 291
12 55 427
24 28 201
16 27 627
14 78 295
19 64 382
16 43 47
6 63 272
3 82 569
17 39 494
13 42 226
12 61 105
25 14 630
25 53 550
5 68 623
18 10 345
17 35 590
9 80 177
16 23 90
25 40 230
15 37 64
16 54 156
30 48 593
23 67 459
26 83 48
2 73 559
24 33 425
25 23 354
18 15 534
31 35 551
0 35 144
10 22 301
0 18 134
15 34 464
27 39 295
6 66 478
27 56 284
8 16 50
28 14 576
20 14 60
10 11 82
11 28 434
16 10 602
9 41 158
16 4 492
11 12 454
18 0 568
24 36 37
34 51 205
13 77 393
7 78 502
31 42 503
18 30 423
17 73 94
11 46 388
10 45 433
9 48 621
22 73 418
15 25 154
0 43 144
20 39 20
1 60 535
15 63 217
10 59 604
15 4 429
12 49 364
15 54 491
10 2 349
22 58 286
19 35 514
20 5 585
14 53 219
7 68 50
19 85 406
9 6 414
4 47 119
4 48 550
22 31 7
16 86 283
0 80 51
9 48 255
29 31 420
30 0 271
18 68 139
12 85 266
5 49 640
10 78 357
22 67 259
15 80 91
10 62 202
31 82 163
31 6 427
13 54 633
8 41 216
2 73 286
1 11 521
1 57 520
0 65 281
31 21 384
6 2 622
19 3 555
18 36 560
22 10 229
4 13 562
20 72 51
7 36 178
12 22 4
19 32 419
32 61 536
2 6 305
29 45 111
6 86 336
21 58 41
19 69 92
34 9 75
4 10 18
1 15 301
27 12 324
17 15 572
23 0 23
5 57 43
23 39 456
22 35 8
10 30 587
7 42 402
23 7 144
5 36 371
10 51 94
15 39 282
5 51 117
17 9 397
20 38 388
29 11 463
1 27 176
16 64 5
34 49 216
29 12 541
18 3 311
31 62 457
30 21 208
21 31 60
9 52 410
12 73 102
29 49 151
34 67 162
22 32 312
22 56 221
2 74 179
34 55 424
3 64 178
19 10 492
11 36 89
2 46 20
28 79 301
17 19 619
15 66 607
5 49 523
21 80 267
26 2 190
29 4 55
22 54 86
0 0 440
25 1 135
34 37 367
9 25 442
20 8 500
23 32 224
33 81 121
7 11 231
18 63 259
5 9 600
15 11 168
15 73 336
33 36 548
25 65 407
17 36 300
18 30 109
27 10 431
33 86 330
13 49 324
27 10 68
16 15 82
27 68 112
2 34 633
27 15 222
0 37 380
0 64 90
13 32 559
20 18 116
10 46 229
11 17 402
14 77 226
34 59 297
25 41 382
14 25 292
34 68 455
16 85 458
20 30 138
17 4 403
23 80 639
24 33 312
6 41 107
20 30 52
34 56 38
15 12 641
16 4 408
5 64 377
30 68 105
14 34 188
5 32 477
20 4 159
18 4 408
9 2 129
4 2 522
4 78 543
12 59 66
17 35 544
30 3 589
5 10 400
19 27 288
30 33 287
0 21 610
25 73 59
18 70 608
23 82 537
12 15 595
29 22 500
23 82 374
15 3 275
14 84 536
1 27 636
28 61 506
6 20 538
4 70 231
22 12 383
0 15 423
6 80 643
8 21 294
23 77 24
26 36 501
18 5 615
24 17 619
25 0 369
8 26 389
34 69 111
