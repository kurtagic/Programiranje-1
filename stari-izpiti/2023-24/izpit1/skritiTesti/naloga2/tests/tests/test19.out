[592, 506, 622, 589, 492, 615, 561, 435, 500, 600, 626, 630, 641, 562, 632, 641, 104, 619, 586, 619, 538, 610, 543, 354, 631, 476, 594, 636, 434, 611, 587, 420, 559, 488, 633, 590, 600, 549, 596, 533, 465, 553, 586, 574, 597, 456, 587, 119, 621, 640, 557, 205, 410, 550, 633, 571, 546, 520, 618, 608, 566, 627, 642, 605, 616, 611, 607, 625, 632, 628, 608, 351, 369, 559, 179, 590, 478, 627, 631, 629, 643, 619, 569, 179, 536, 484, 582]
